partcad/src/partcad/adhoc/convert.py

```
@@ -0,0 +1,86 @@
+
+    # Generate a temporary configuration file for PartCAD
+    temp_dir = Path(tempfile.mkdtemp())  # Create a temporary directory for PartCAD config
+    try:
+        generate_partcad_config(temp_dir, input_type, input_path)
+
+        # Initialize PartCAD context and load the project
+        ctx = Context(root_path=temp_dir, search_root=False)
+        project = ctx.get_project("/")
+        part = project.get_part("input_part")
+        if not part:
+            raise RuntimeError("Failed to load the input part.")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)